tasks/sysv.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: create directories
+  become: yes
+  become_user: root
+  with_items:
+    - { d: '{{couchdb_dir}}/var/run', m: '0700' }
+  file: >-
+    path={{item.d}}
+    state=directory
+    owner={{couchdb_user}}
+    group={{couchdb_group}}
+    mode={{item.m}}
+
+- name: install init.d service
+  become: yes
+  become_user: root
+  register: couchdb_service_modified
+  with_items:
+    - etc/init.d/couchdb2
```

Comment:
I like to parameterize with the file and target directory as separate parts of the ```item```.
```yml
with_items:
  - f: couchdb2
    d: /etc/init.d
```
and move all of the templates into the top level ```templates``` directory.